src/main/java/roboy/dialog/personality/SmallTalkPersonality.java

```diff
@@ -35,30 +39,12 @@
     public SmallTalkPersonality(Verbalizer verbalizer) {
         this.verbalizer = verbalizer;
 
-
-        List<String> nameQuestions = Arrays.asList(
-                "How can I call you?",
-                "What is your name?",
-                "Who are you?");
-        List<String> occupationQuestions = Arrays.asList(
-                "What do you do?",
-                "What is your profession?");
-        List<String> originQuestions = Arrays.asList(
-                "Where are you from?",
-                "Where were you born?",
-                "I have hard time guessing your home country. What is it?",
-                "Which town do you call your home?");
-        List<String> hobbyQuestions = Arrays.asList(
-                "What is your favorive thing to do?",
-                "What is your hobby?",
-                "What do you do in your free time?",
-                "How do you spend your free time?",
-                "What is your favorive pasttime activity?");
-        List<String> moviesQuestions = Arrays.asList(
-                "What is your favourite movie?",
-                "What was the last movie you saw in the cinema?",
-                "What is your favorite TV show?",
-                "Which comedy do you like the most?");
+        String path = "ontology/questions/";
+        List<String> nameQuestions = getQuestionFromJsonFile(path + "nameQuestion.json");

```

Comment:
you created a separate class Question which is literally list of strings and you used it once? why are you not using it here?